docker-compose.yml

```diff
@@ -556,7 +552,7 @@ services:
     # Connect to the custom default network
     networks:
     - network
-
+    
```

Comment:
Please don't add/leave trailing whitespaces. Use an editor that [highlights them](https://marketplace.visualstudio.com/items?itemName=shardulm94.trailing-spaces).

Reply:
done, thanks for the tip. I was pretty careless with that one.